backend/migration.ts

```diff
@@ -1,10 +1,10 @@
-import { Umzug, SequelizeStorage } from 'umzug';
-import { chain, each, map, last } from 'lodash';
-import { DataTypes } from 'sequelize';
+import { chain, each, last, map } from 'lodash';
+import type { QueryInterface, Sequelize, DataTypes } from 'sequelize';
```

Comment:
I'm not sure about `DataTypes` here. 
I suspect (I'm not 100% sure) it was imported with just `import` (and not `import type`) intentionally.

Reply:
I was surprised here as well. It is used with `typeof` operator (and it's the only usage), so I would expect a need to import it as a value. But this is not the case, looks like in such cases `import type` is sufficient and is actually enforced by the eslint rule. Everything compiles fine so I assume we are good here.

Reply:
OK, so if it compiles fine, then are we sure that:

1. TS checks (`check-types`) in Stage and Composer TS are not failing?
2. we get proper support from IDE with DataTypes? (see video below)

https://user-images.githubusercontent.com/5202105/166656866-c976d5a5-5197-464e-aeea-ace41ba177ec.mp4

Reply:
I verified both points - no issues.

Reply:
Thanks! So we are good to go 👍🏻